### PR TITLE
Updated README.md and README

### PR DESCRIPTION
**Pull request by joachimmetz**
Updated README.md and README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,20 @@
 
 *super timeline all the things*
 
-In short, plaso is a Python-based backend engine for the tool 
-[log2timeline](http://plaso.kiddaland.net "Plaso home of the super timeline").
+Plaso (Plaso Langar Að Safna Öllu) is the Python-based engine used by [tools](https://plaso.readthedocs.io/en/latest/sources/user/Users-Guide.html#the-tools)
+for automatic creation of timelines. Plaso its default behavior is to create
```
Comment (rgayon) [inline README.md:6]: -its

File: README.md [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
+super timelines but it also supports creating more [targeted timelines](http://blog.kiddaland.net/2013/02/targeted-timelines-part-i.html).
 
-### A longer version
+These timelines support digital forensic investigators/analysts, to correlate
+the large amount of information found in logs and on an average computer.
```
Comment (rgayon) [inline README.md:10]: Why just talk about logs here?

File: README.md (hunk 1 above)
Comment (joachimmetz) [reply README.md:6]: ack

File: README.md (hunk 2 above)
Comment (joachimmetz) [reply README.md:10]: `logs and on an average` but I'll rephrase

**Comment by joachimmetz** [comment]
@rgayon PTAL

**Pull request merged**